apps/src/sites/code.org/pages/public/students.js

```diff
@@ -0,0 +1,16 @@
+import $ from 'jquery';
+
+import statsigReporter from '@cdo/apps/metrics/StatsigReporter';
+
+$(document).ready(() => {
+  const isInSignupExperiment = statsigReporter.getIsInExperiment(
+    'new_sign_up_v1',
+    'showNewFlow',
+    false
+  );
+  const signupLink = document.getElementById('sign-in-button');
```

Comment:
Do we want to redirect a sign-in link to sign-up? Or is this one of the spots we've mislabeled it previously?

Reply:
Unfortunately this was just previously mislabeled! I imagine there is some sort of styling tied to this ID but I can go track it down and update it to reflect what the button actually does.

Reply:
No worries! I just wanted to double check just in case!

Reply:
Maybe if possible could we add a little comment there just to make a note of it being mislabeled in case anyone else does future work there please?

Reply:
Because this file is going to disappear as soon as the ab test is over, I added the comment to the student.haml file itself!

Reply:
Oh good point good point, thank you!